pretty_gpx/city/city_drawing_figure.py

```diff
@@ -29,6 +29,12 @@ class CityDrawingFigure(BaseDrawingFigure):
     road_data: list[BaseDrawingData]
```

Comment:
@JulesL2 
Indeed I agree that the left map looks better (eventhough we still need to play a bit with the colors). Especially for small runs in the countryside :ok_hand: :smiley: 
Ideally there will be an option in the GUI to choose what we're displaying

However, I'm still not convinced by the airport and the railways  :/

![image](https://github.com/user-attachments/assets/735b80da-9232-48a5-9887-5fda90b3335f)